libmscore/textbase.cpp

```diff
@@ -888,6 +892,8 @@ QString TextBlock::remove(int column)
                         if (i->text.isEmpty())
                               _fragments.erase(i);
                         simplify();
+                        if(this->fragments().size() == 0) // add an empty fragment, because this is where the formatting is held
```

Comment:
Sir, you added the same 2 lines in 4 places! Could you please create a private method in `TextBlock` class called `void insertEmptyFragmentIfNeeded(TextCursor* cursor)` and put those 2 strings in it.

After that, you will need to replace those 2 strings in your PR with a call of this method.

And please remove `this`. It is not needed.

Reply:
Oops, I changed the insertEmptyFragment method for the other related PR and did not adjust accordingly, I'll fix it right away :-).

Reply:
Done.